Makefile

```diff
@@ -1,32 +1,67 @@
-
 export CGO_ENABLED:=0
-LD_FLAGS="-w -X github.com/coreos/matchbox/matchbox/version.Version=$(shell ./git-version)"
-LOCAL_BIN=/usr/local/bin
 
-all: build
-build: clean bin/matchbox bin/bootcmd
+VERSION=$(shell ./scripts/git-version)
+LD_FLAGS="-w -X github.com/coreos/matchbox/matchbox/version.Version=$(VERSION)"
 
-tools:
-	./scripts/gentools
+REPO=github.com/coreos/matchbox
+IMAGE_REPO=coreos/matchbox
+QUAY_REPO=quay.io/coreos/matchbox
 
-codegen: tools
-	./scripts/codegen
+all: build
+
+build: clean bin/matchbox bin/bootcmd
 
 bin/matchbox:
-	go build -o bin/matchbox -ldflags $(LD_FLAGS) -a github.com/coreos/matchbox/cmd/matchbox
+	@go build -o bin/matchbox -v -ldflags $(LD_FLAGS) $(REPO)/cmd/matchbox
```

Comment:
You might want to use a variable instead of `@`. That way, you can turn on or off verbose building. Check out https://github.com/crawford/arm-toolkit/blob/be1984fabddaeff4777ff96b7f6a5830ee760102/makefile.mk#L40 as an example.